frontend/interfaces/policy.ts

```diff
@@ -89,7 +88,7 @@ export interface IPolicyFormData {
   platform?: SelectedPlatformString;
   name?: string | number | boolean | undefined;
   query?: string | number | boolean | undefined;
-  team_id?: number;
+  team_id?: number | null;
```

Comment:
samesies team_id null is shown in the new API https://github.com/fleetdm/fleet/pull/17227/files currently line 6074

Reply:
We can update this to remove the `?` too, I think.

Reply:
Oh yeah, I tried, but apparently for global we don't actually send the key `team_id` to the API, so maybe I need to remove the null instead, unless it's there on edit. >.<